backend/core/src/email/email.service.ts

```diff
@@ -415,4 +434,32 @@ export class EmailService {
       throw new HttpException("email failed", 500)
     }
   }
+
+  async sendCSV(user: User, listingName: string, listingId: string, applicationData: string) {
+    void (await this.loadTranslations(
+      user.jurisdictions?.length === 1 ? user.jurisdictions[0] : null,
+      user.language || Language.en
+    ))
+    const jurisdiction = await this.getUserJurisdiction(user)
+    await this.send(
+      user.email,
+      jurisdiction.emailFromAddress,
+      `${listingName} applications export`,
+      this.template("csv-export")({
+        user: user,
+        appOptions: { listingName, appUrl: this.configService.get("PARTNERS_PORTAL_URL") },
+      }),
+      undefined,
+      {
+        data: applicationData,
+        name: `applications-${listingId}-${this.createDateStringFromNow()}.csv`,
+        type: "text/csv",
+      }
+    )
+  }
+
+  createDateStringFromNow(format = "YYYY-MM-DD_HH:mm:ss"): string {
+    const now = new Date()
+    return dayjs(now).format(format)
+  }
```

Comment:
I'm ok with keeping this here, but would the formatLocalDate function work for this use case?